commit d424b93040a1101b99841707a53093af45f0a7dd
Author: Pi-Hsun Shih <peter50216@gmail.com>
Date:   Fri Apr 3 22:31:04 2015 +0800

    fix ruby gets problem.

diff --git a/autoload/rubycomplete.vim b/autoload/rubycomplete.vim
index df7ef5c..b44fbe3 100644
--- a/autoload/rubycomplete.vim
+++ b/autoload/rubycomplete.vim
@@ -703,8 +703,9 @@ class VimRubyCompletion
 
         cv = eval("self.class.constants")
         vartype = get_var_type( receiver )
+
         dprint "vartype: %s" % vartype
-        if vartype != ''
+        if !vartype.empty? && vartype != 'gets'
           load_buffer_class( vartype )
 
           begin
@@ -732,7 +733,7 @@ class VimRubyCompletion
             methods.concat m.instance_methods(false)
           }
         end
-        variables += add_rails_columns( "#{vartype}" ) if vartype && vartype.length > 0
+        variables += add_rails_columns( "#{vartype}" ) if vartype && vartype.length > 0 && vartype != 'gets'
 
       when /^\(?\s*[A-Za-z0-9:^@.%\/+*\(\)]+\.\.\.?[A-Za-z0-9:^@.%\/+*\(\)]+\s*\)?\.([^.]*)/
         message = $1
